application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -204,7 +205,13 @@ adminTools.dashboard.healthcheck.modal.wikiBins.header.page=Deleted pages
 adminTools.dashboard.healthcheck.modal.wikiBins.header.attach=Deleted attachments
 adminTools.dashboard.healthcheck.modal.wikiBins.header.total=Total
 adminTools.dashboard.healthcheck.modal.wikiBins.documents=View deleted pages
-adminTools.dashboard.healthcheck.modal.wikiBins.attachments=View deleted attachments</content>
+adminTools.dashboard.healthcheck.modal.wikiBins.attachments=View deleted attachments
+adminTools.dashboard.instanceUsage.modal.wikiSize.attachmentsCount=Attachments count
+adminTools.dashboard.instanceUsage.modal.wikiSize.attachmentsSize=Attachments size
```

Comment:
This is the total size for attachments in that wiki, right?

Reply:
Yes, it represents the total size of the attachments in a wiki. Maybe replace it with `Total attachments size` if you think it's not clear enough?